crates/hyperqueue/src/client/commands/job.rs

```diff
@@ -67,11 +69,12 @@ pub struct JobForgetOpts {
 
 #[derive(Parser)]
 pub struct JobTaskIdsOpts {
-    /// Single ID, ID range or `last` to display the most recently submitted job
+    /// Selects job(s)
```

Comment:
We should provide some unified help (same as we do with `duration_doc!`) for the selector syntax. Especially if we're not going to enumerate the supported selector variants, because it differs between different commands.